packages/subgraph/src/packages/multisig/multisig.ts

```diff
@@ -152,15 +170,17 @@ export function handleProposalExecuted(event: ProposalExecuted): void {
 export function handleMembersAdded(event: MembersAdded): void {
   const members = event.params.members;
   for (let index = 0; index < members.length; index++) {
-    const member = members[index].toHexString();
-    const pluginId = event.address.toHexString();
-    const memberId = pluginId + '_' + member;
+    const memberAddress = members[index];
+    const pluginEntityId = generatePluginEntityId(event.address);
+    const memberEntityId = [pluginEntityId, memberAddress.toHexString()].join(
+      '_'
+    );
```

Comment:
I noticed we create the member ID for multisig and Tokenvoting using the pluginId & members's address

however i see the order of the 2 paramsare oposite to each other in each one,

can u investiage what would be the implication of unifying them on SDK, App ...etc

and if feel it should not be part this PR, please create a follow up task

Reply:
Okay, in the sdk will have no implications on the app i have no idea, will ask them

Reply:
Asked @sepehr2github and seem ok on their side so It seems okay